test/OpenTelemetry.Tests/Concurrency/MetricsConcurrencyTests.cs

```diff
@@ -0,0 +1,63 @@
+// <copyright file="MetricsConcurrencyTests.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using Microsoft.Coyote;
+using Microsoft.Coyote.SystematicTesting;
+using OpenTelemetry.Metrics.Tests;
+using Xunit;
+
+namespace OpenTelemetry.Tests.Concurrency;
+
+public class MetricsConcurrencyTests : AggregatorTestsBase
```

Comment:
This looks odd. Use composition over inheritance.

Reply:
updated; thanks!